src/coreclr/debug/inc/dbgtargetcontext.h

```diff
@@ -466,7 +472,16 @@ typedef DECLSPEC_ALIGN(16) struct {
     /* +0x338 */ DWORD64 Bvr[DT_ARM64_MAX_BREAKPOINTS];
     /* +0x378 */ DWORD Wcr[DT_ARM64_MAX_WATCHPOINTS];
     /* +0x380 */ DWORD64 Wvr[DT_ARM64_MAX_WATCHPOINTS];
-    /* +0x390 */
+
+    //
+    // Sve Registers
+    //
+    //TODO-SVE: Support Vector register sizes >128bit
+    /* +0x390 */ DWORD Vl;
+    /* +0x3a0 */ SVE128 Z[32];
```

Comment:
This should be DT_SVE128, that's what's causing the Windows arm64 build to fail.